monitoring/grafana-dashboard/host-metrics.json

```diff
@@ -942,6 +942,10 @@
   "templating": {
     "list": []
   },
+  "time": {
+    "from": "now-5m",
```

Comment:
Defaulting to watching only the last 5 minutes doesn't make any sense? A node operator is interested in at least the last few hours, this should be something like `now-12h`.